init/lib/stop.go

```diff
@@ -23,36 +23,57 @@ import (
 	"github.com/pkg/errors"
 )
 
-func StopProcess(process *os.Process) error {
-	if err := process.Signal(syscall.SIGTERM); err != nil {
-		if !strings.Contains(err.Error(), "os: process already finished") {
-			return errors.Wrap(err, "failed to stop process")
+func StopService(procs []*os.Process) error {
```

Comment:
Keeping the names here both means we don't need to create a slice from our named process map, and also means we can be more specific in our logging than just "a" process failed to stop